### ISSUE TITLE
@azure/msal-angular: peerDependencies operators don't imply support for Angular versions 6?

### ISSUE THREAD

**Issue by RobARichardson** [AzureAD/microsoft-authentication-library-for-js#398]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[x] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Library version
@azure/msal-angular 0.1.1

#### Documentation for Msal-Angular says:
"...The MSAL library preview for Angular is a wrapper of the core MSAL.js library which enables Angular(**4.3 and above**)..."

#### However, peerDependencies imply the library only supports Angular v4.x:
"peerDependencies": {
    "@angular/common": "^4.3.0",
    "@angular/core": "^4.3.0"
  }

#### This causes errors when using 'ng update' (Angular CLI) w/ Angular versions higher than 4:
```
Package "@azure/msal-angular" has an incompatible peer dependency to "@angular/core" (requires "^4.3.0" (extended), would install "6.1.3").
Package "@azure/msal-angular" has an incompatible peer dependency to "@angular/common" (requires "^4.3.0" (extended), would install "6.1.3").
```

## Expected behavior
If msal-angular indeed supports Angular 5 & 6, no warnings should show up for @azure/msal-angular when using 'ng update' w/ those versions of Angular.

**Comment by nehaagrawal** [comment]
@RobARichardson  There is no 'ng upgrade' command. I think you meant 'ng update'? Can you please tell me what was your local angular version when you ran the 'ng update'?

**Comment by RobARichardson** [comment]
@nehaagrawal yes, I meant 'ng update'.  I've edited my original post to correct the typo.

I'm using Angular 6.1.3

**Comment by nehaagrawal** [comment]
@RobARichardson  I am going to release another version with a dependency/peer-dependency on angular 5.0. However you can  use the current msal-angular version with angular 5.0.  It works. Please check our dev samples. We are using angular 5 there. I am going to update the documentation that we currently support only 4.3.* to 5.*.*.

**Comment by RobARichardson** [comment]
@nehaagrawal okay.  Yeah, I noticed that the sample isn't up to Angular 6 yet.  Any ideas on when support for 6 will happen and what is preventing it?  We would really like to use msal-angular but can't downgrade to Angular 5.

**Comment by nehaagrawal** [comment]
@RobARichardson  On another thread I saw that you had some success by using rxjs-compat for rxjs breaking changes. Didn't that work?

**Comment by RobARichardson** [comment]
@nehaagrawal yeah, using rxjs-compat worked great, but then I ran into the issues related to #386.  Barring any other issues, I'll still try to use msal-angular in my Angular 6 project even though the library doesn't officially support it yet.

**Comment by nehaagrawal** [comment]
@RobARichardson  I have a fix for issue #386. I am ill releasing it in this week.  You can still use msal-angular with angular 5. We don't have support for angular 6 yet but that's in our backlog and we will work on it in future.

**Comment by nehaagrawal** [comment]
closing this issue in favor of  https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/421. Please track there for future updates.
